apps/wallet/src/ui/app/shared/input/password/validation.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as Yup from 'yup';
+
+function containsUpperCaseLetters(str: string) {
+    for (const letter of str) {
+        if (
+            letter.toLowerCase() !== letter.toUpperCase() &&
+            letter === letter.toUpperCase()
+        ) {
+            return true;
+        }
+    }
+    return false;
+}
+
+export const passwordValidation = Yup.string()
```

Comment:
We should talk through what we want the actual password requirements to be. We talked about using a password strength checker (cc @joyqvq)

Reply:
Yeah Joy mentioned it to me but for now I just did what the design had. I added it to the list [here](https://github.com/MystenLabs/sui/issues/4926)

Reply:
Yes. Thanks. 
The UI copy is meant to be a placeholder until we are figuring out the requirements.
Do we have them @Jordan-Mysten ?